David

davidjohnsson96 opened the pull request.

davidjohnsson96 committed:
add upload without load_button. load_button still there but not visible

davidjohnsson96 committed:
Add radio_buttons and collapse to asa widget

davidjohnsson96 committed:
Update filtering and search_result to adapt to asa widget changes

davidjohnsson96 committed:
Add working layout

davidjohnsson96 committed:
Add working layout

raoulkent commented on components/active_filters.py:
```diff
@@ -8,17 +8,17 @@ class ActiveFilters:
     @staticmethod
     def get_component():
         widget = dbc.Col(
+            width={"width": "60%"},
```
Can be replaced with `className="col-6"`or another such bootstrap class 😄

raoulkent replied:
Still ^

raoulkent commented on components/notes.py:
```diff
@@ -7,7 +7,10 @@ class Notes:
         widget = dbc.Col(
-            style={"max-width": "40%"},
+            style={
+                "width": "40%",
```
Can be replaced with `className="col-4"` or another such bootstrap class

raoulkent replied:
Still ^

raoulkent commented on components/content.py:
```diff
@@ -15,7 +15,9 @@ def get_component():
                 html.H1(id="h1", children="Plando-prototype"),
                 dbc.Row(
-                    children=[ActiveFilters.get_component(), Notes.get_component()]
+                    style={"width": "60vw"},
```
Can be replaced with `className="col-6"` or another such bootstrap class

raoulkent replied:
Still ^

raoulkent commented on components/sidebar_toggle.py:
```diff
@@ -50,7 +50,17 @@ def add_callback(app):
             SidebarToggle.is_open = not SidebarToggle.is_open
             return (
-                {"display": "block"} if SidebarToggle.is_open else {"display": "none"},
+                {
```
This is fixed in f18392d223cf67d3e00cdae9d44df1f4ee3edfdb. So merging sidebar-toggler-refactor into this branch should be enough.

davidjohnsson96 committed:
update layout

davidjohnsson96 committed:
update layout and add callback to notes

linneahallin committed:
Change stat_code from dropdown to checklist

linneahallin committed:
Remove dcc import

davidjohnsson96 committed:
Update datatable styling

HodgesDexner committed:
Improve and refactor styling

Does: Moved styling of components to custom.css. Changed color to
correct color scheme. Corrected padding and margin on active filters and
result table.

davidjohnsson96 committed:
Merge branch 'statistics_code' of github.com:GideonJagen/Kandidatarbete into David

davidjohnsson96 committed:
Merge branch 'Johan' of github.com:GideonJagen/Kandidatarbete into David

davidjohnsson96 committed:
Update layout

raoulkent commented on components/content.py:
```diff
@@ -15,7 +15,9 @@ def get_component():
                 dbc.Row(
-                    children=[ActiveFilters.get_component(), Notes.get_component()]
+                    style={"width": "60vw"},
+                    className="row p-1 justify-content-start",
```
Change `className="row p-1 justify-content-start",`
To `className="justify-content-start",`

raoulkent commented on components/sidebar_toggle.py:
```diff
@@ -50,7 +50,17 @@ def add_callback(app):
-                {"display": "block"} if SidebarToggle.is_open else {"display": "none"},
+                {
+                    "display": "block",
+                    "height": "inherit",
```
These height and inherit stylings were moved in a3e7a1c. Those changes seem to have been lost in the merge.

raoulkent commented on components/reset_filter_button.py:
```diff
@@ -7,7 +7,7 @@ def get_component():
-            color="danger",
-            style={"margin-left": "50em", "height": "40px"},
+            color="link",
+            style={"height": "2.5em"},
```
I think there's a bootstrap class for making the button size larger.
Maybe try `className="btn-lg"`?

raoulkent commented on main.py:
```diff
@@ -40,7 +41,12 @@
             ],
-            style={"margin-right": "0", "margin-left": "0", "height": "100%"},
+            style={
+                "margin-right": "0",
```
These margin left/right should be replaced with bootstrap classes:
`className="ml-0 mr-0",`

raoulkent reviewed (changes requested):
Found a few things that could be changed.

One was a loss of information during merge, and most others are that there are existing bootstrap tools that we can use instead of hard-coded styling!

davidjohnsson96 committed:
update pr

davidjohnsson96 committed:
update pr

raoulkent reviewed (approved):
Beri gud
